pom.xml

```diff
@@ -44,7 +44,7 @@
     <dep.slf4j.version>1.7.6</dep.slf4j.version>
     <dep.guava.version>18.0</dep.guava.version>
     <forkCount>2</forkCount>
-    <parquet.version>1.8.1-drill-r0</parquet.version>
+    <parquet.version>1.10.0</parquet.version>
```

Comment:
1.10? Is it safer to upgrade to 1.8.3 and then test out 1.9/1.10 before upgrading to it?
Also, with this change the Hive parquet version is 1.8.2. I wonder what impact that might have on compatibility?

Reply:
`1.8.3` as well as `1.8.1-drill-r0` are supposed to be a patch release on top of `1.8.0`. Unfortunately `parquet-mr` does not properly follow semantic versioning and have functional and API level changes in the patch releases. On top of that, both `1.9.0` and `1.10.0` are not backward compatible with `1.8.0` and/or `1.8.3`, so upgrade to `1.8.3` will not help us with the upgrade to `1.10.0` later either or make it safer. I'd suggest to pay the price once. Additionally, the latest parquet version may have a functionality to be used in filter pushdown. @arina-ielchiieva what is your take?

Parquet libraries used by hive are shaded within `drill-hive-exec-shaded`, so hive is guarded from the parquet-mr library upgrade.

Reply:
Well, I need this upgrade to implement varchar filter push down, since it has been fixed in 1.10.0 but not in 1.8.3. I think if you unit tests pass, along with Functional & Advanced we are safe to go.

Reply:
Fair enough.